ClientCore/PlatformShim/EncodingExt.cs

```diff
@@ -0,0 +1,16 @@
+using System.Text;
+
+namespace ClientCore.PlatformShim;
+
+public static class EncodingExt
+{
+    static EncodingExt()
+    {
+#if !NETFRAMEWORK
+        Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+#endif
+        ANSI = Encoding.GetEncoding(0);
+    }
+
+    public static Encoding ANSI { get; }
+}
```

Comment:
This is for supporting the legacy ANSI encoding (not Windows 2052, and also not any specific encoding, in case non-Asian people misunterstand again) as FinalSun/FinalAlert read the configuration in ANSI